### PR TITLE
Ft 162953296 users meetup able view questions

### PR DESCRIPTION
**Pull request by b0nbon1**
*What does this PR do?*

This is a PR enables user to view posted questions

*Description of Task to be completed?*

The following are the tasks to be completed:

• Implement creates specific meet up with list of question

*How to manually test feature?*

1. clone this repo  
2. cd in folder  

3 switch to feature branch  
`Ft-162953296-users-meetup-able-view-questions`

4. Open `questions.html` in your browser

**screenshots**
screenshot from 2019-01-05 19-53-49

### CODE REVIEW

**Comment by Ogollah** [comment]
This is great.

File: UI/css/questions.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    background: red;
+  }
+  
+  .counter {
+    margin-top: 20px;
+    padding: 10px;
+    border: 2px solid grey;
+    display: inline-block;
+    vertical-align: top;
+  }
```
Comment (Ogollah) [inline UI/css/questions.css]: needs a new line here

File: UI/templates/meetup.html [hunk 2]
```diff
@@ -90,7 +90,6 @@ <h2>Details</h2>
 </section>
 <footer>
     <p>Questioner, Copyright &copy; 2019</p>
-</footer>
-<script type="text/javascript" src="../js/date.js"></script>   
+</footer>  
 </body>
 </html>
```
Comment (Ogollah) [inline UI/templates/meetup.html]: new line

File: UI/templates/question.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+</body>
+</html>
+
+
+<footer>
+    <p>Questioner, Copyright &copy; 2019</p>
+</footer>
+   
+</body>
+</html>
```
Comment (Ogollah) [inline UI/templates/question.html]: needs a new line

Comment (b0nbon1) [reply UI/templates/question.html]: ah what do you mean by newline?

Comment (Ogollah) [reply UI/templates/question.html]: You need a new line with no code at the end of your file.

**Pull request merged**